func_ptr_alias regressions failing on OSX

The four `test/basic/func_ptr_alias*.c` regressions fail (with `UNKNOWN` result) on OS X.

The cause is easy to determine:
````
$ smack test/basic/func_ptr_alias.c 
SMACK program verifier version 1.8.0
test/basic/func_ptr_alias.c:9:30: error: only weak aliases are supported on darwin
int incr(int) __attribute__((alias ("__incr")));
                             ^
1 error generated.
Traceback (most recent call last):
  File "/usr/local/bin/smack", line 13, in <module>
    smack.top.main()
  File "/usr/local/bin/../share/smack/top.py", line 646, in main
    frontend(args)
  File "/usr/local/bin/../share/smack/top.py", line 270, in frontend
    return frontends()[lang](args)
  File "/usr/local/bin/../share/smack/top.py", line 328, in clang_frontend
    try_command(compile_command + ['-o', bc, c], console=True)
  File "/usr/local/bin/../share/smack/utils.py", line 69, in try_command
    raise Exception(output)
Exception: test/basic/func_ptr_alias.c:9:30: error: only weak aliases are supported on darwin
int incr(int) __attribute__((alias ("__incr")));
                             ^
1 error generated.
````

--- a/test/basic/func_ptr_alias.c
+++ b/test/basic/func_ptr_alias.c
@@ -6,7 +6,11 @@ int __incr(int x) {
   return x + 1;
 }
 
+#ifdef __MACH__
+int (*incr)(int) = __incr;
+#else
 int incr(int) __attribute__((alias ("__incr")));
+#endif
 
 int main(void) {
   int x = __VERIFIER_nondet_int();

--- a/test/basic/func_ptr_alias1.c
+++ b/test/basic/func_ptr_alias1.c
@@ -10,8 +10,13 @@ int __decr(int x) {
   return --x;
 }
 
+#ifdef __MACH__
+int (*incr)(int) = __incr;
+int (*decr)(int) = __decr;
+#else
 int incr(int) __attribute__((alias ("__incr")));
 int decr(int) __attribute__((alias ("__decr")));
+#endif
 
 int main(void) {
   int (*fp)(int);

--- a/test/basic/func_ptr_alias1_fail.c
+++ b/test/basic/func_ptr_alias1_fail.c
@@ -10,8 +10,13 @@ int __decr(int x) {
   return --x;
 }
 
+#ifdef __MACH__
+int (*incr)(int) = __incr;
+int (*decr)(int) = __decr;
+#else
 int incr(int) __attribute__((alias ("__incr")));
 int decr(int) __attribute__((alias ("__decr")));
+#endif
 
 int main(void) {
   int (*fp)(int);

--- a/test/basic/func_ptr_alias_fail.c
+++ b/test/basic/func_ptr_alias_fail.c
@@ -6,7 +6,11 @@ int __incr(int x) {
   return x + 2;
 }
 
+#ifdef __MACH__
+int (*incr)(int) = __incr;
+#else
 int incr(int) __attribute__((alias ("__incr")));
+#endif
 
 int main(void) {
   int x = __VERIFIER_nondet_int();

